android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ConfigService.kt

```diff
@@ -103,10 +106,9 @@ interface ConfigService {
           // TODO: Does not support multi organization yet,
           // https://github.com/opensrp/fhircore/issues/1550
           ConfigurationRegistry.ORGANIZATION ->
-            paramsMap
-              ?.get(SharedPreferenceKey.PRACTITIONER_DETAILS_ORGANIZATION_IDS.name)
-              ?.firstOrNull()
-              ?.substringAfter("/")
+            mandatoryTags
+              .firstOrNull { it.display.contentEquals(SyncStrategy.ORGANIZATION.value) }
```

Comment:
Should we ignore case here when comparing the texts?

Reply:
Let's ignore case, just to make sure